waspc/src/NpmDependency.hs

```diff
@@ -0,0 +1,21 @@
+module NpmDependency
```

Comment:
Yes, because it is more of a standalone thing, a concept. For example, in WebAppGenerator.hs, we define them directly -> it wouldn't make much sense for them to be part of AST, at least I think so. I am also not 100% sure this is the best way, but it felt like good enough guess for now.